aiida/cmdline/params/options/__init__.py

```diff
@@ -451,7 +452,43 @@ def decorator(command):
 
 FREQUENCY = OverridableOption('-F', '--frequency', 'frequency', type=click.INT)
 
-VERBOSE = OverridableOption('-v', '--verbose', is_flag=True, default=False, help='Be more verbose in printing output.')
+VERBOSE = OverridableOption('-v', '--verbose', count=True, help='Print more verbose output.')
+
+
+def VERBOSITY(logger='cli', *names, **kwargs):  # pylint: disable=keyword-arg-before-vararg,invalid-name
```

Comment:
So far this module has been reserved for ready to use `OverridableOptions` where everything could be changed except for the flag names. Would it make sense to put this command in its own file and then define the `VERBOSITY` overridable option here based on that, where everything can still be configured except for the flag names?

Reply:
I definitely agree that the option doesn't belong here and should go in a separate file.
I wonder whether we need to provide an overridable option for it... I don't see where it would be needed, but perhaps it's good to have?

Reply:
This way plugins can reuse it while automatically making sure they use the same flags for consistency. Of course, saying that, I am thinking that if we suggest plugins to reuse these options (I certainly do in mine) we can maybe also no longer just delete them if we don't use them in `aiida-core` anymore.

Reply:
Done!